templates/dashboard/student-dashboard.php

```diff
@@ -0,0 +1,263 @@
+				<h3 class="tutor-stat-card-title">
+					<?php echo esc_html__( 'Time Spent', 'tutor' ); ?>
+				</h3>
+				<div class="tutor-stat-card-icon">
+					<?php tutor_utils()->render_svg_icon( Icon::TIME, 20, 20 ); ?>
+				</div>
+			</div>
+			<div class="tutor-stat-card-content">
+				<div class="tutor-stat-card-value">
+					375h+
+				</div>
+				<p class="tutor-stat-card-change">
+					+2 this month
+				</p>
+			</div>
+		</div>
+	</div>
+
+	<!-- @TODO:: Need to update this with dynamic data -->
+	<div x-data="tutorModal({ id: 'tutor-time-spent-modal' })" x-cloak>
+		<template x-teleport="body">
+			<div x-bind="getModalBindings()">
+				<div x-bind="getBackdropBindings()"></div>
+				<div x-bind="getModalContentBindings()" style="width: 354px;">
+					<div class="tutor-modal-body tutor-px-9 tutor-pt-9 tutor-pb-8 tutor-text-center">
+						<?php tutor_utils()->render_svg_icon( Icon::CONFETTI, 32, 32, array( 'class' => 'tutor-icon-exception2' ) ); ?>
+
+						<h3 class="tutor-h3 tutor-mb-2 tutor-mt-6"><span class="tutor-font-regular">Fantastic,</span> Johny!</h3>
+
+						<div class="tutor-tiny tutor-text-secondary tutor-mb-6">
+							<?php echo esc_html__( "You've dedicated over", 'tutor' ); ?>
+						</div>
+
+						<h2 class="tutor-h2 tutor-text-exception4 tutor-py-6 tutor-surface-l3 tutor-rounded-lg tutor-mb-6">375+ hours</h2>
+
+						<p class="tutor-p2 tutor-mb-7">That's <span class="tutor-font-medium">1,350,000</span> minutes, and <span class="tutor-font-medium">81,000,000</span> seconds! Incredible!</p>
+
+						<button 
+							class="tutor-btn tutor-btn-primary tutor-btn-large tutor-rounded-full tutor-btn-block tutor-gap-2" 
+							@click="TutorCore.modal.closeModal('tutor-time-spent-modal')"
+						>
+							<?php tutor_utils()->render_svg_icon( Icon::HAPPY, 20, 20 ); ?>
+							<?php esc_html_e( "I'm Happy", 'tutor' ); ?>
+						</button>
+					</div>
+				</div>
+			</div>
+		</template>
+	</div>
+</div>
+
+<?php
+/**
+ * Active users in progress courses
+ */
+$placeholder_img     = tutor()->url . 'assets/images/placeholder.svg';
+$courses_in_progress = CourseModel::get_active_courses_by_user( get_current_user_id(), 0, 2 );
+?>
+
+<?php if ( $courses_in_progress && $courses_in_progress->have_posts() ) : ?>
+	<div class="tutor-student-dashboard-courses">
+		<div class="tutor-flex tutor-items-center tutor-justify-between tutor-mb-4">
+			<div class="tutor-small tutor-font-medium">
+				<?php esc_html_e( 'Continue Learning', 'tutor' ); ?>
+			</div>
+			<a 
+				href="<?php echo esc_url( tutor_utils()->tutor_dashboard_url( 'enrolled-courses' ) ); ?>" 
+				class="tutor-btn tutor-btn-link tutor-btn-x-small tutor-text-brand tutor-p-none tutor-min-h-0"
+			>
+				<?php esc_html_e( 'See All', 'tutor' ); ?>
+			</a>
+		</div>
+		<div class="tutor-flex tutor-flex-column tutor-gap-4">
+		<?php
+		while ( $courses_in_progress->have_posts() ) :
+			$courses_in_progress->the_post();
+			$tutor_course_img  = get_tutor_course_thumbnail_src();
```

Comment:
This method already has the placeholder fallback `get_tutor_course_thumbnail_src`. We don't need to use it explicitly.